clang/utils/TableGen/ClangOpenCLBuiltinEmitter.cpp

```diff
@@ -1,4 +1,4 @@
-//===- ClangOpenCLBuiltinEmitter.cpp - Generate Clang OpenCL Builtin handling
+//===- ClangProgModelBuiltinEmitter.cpp - Generate Clang Builtin handling
```

Comment:
The file should be renamed accordingly.

Reply:
I agree, I'll do that once approved if you don't mind. I was trying to make sure the changes in the file remains clearly visible (renaming the file may affect the way github display it).

Reply:
Here GitHub was able to handle both renaming + content changes: https://github.com/intel/llvm/pull/1306/files#diff-e4c028e8ba11e842d27a835ad03e47be.
Can we give it a try?
You can do it in a separate commit and revert if it doesn't work as expected.

Reply:
TIL it actually works in github :)